R/utility.R

```diff
@@ -1,35 +1,46 @@
-getInstallationCommand <- function(packages){
-  installation <- ""
-
-  for(package in packages){
-    installation <- paste0(installation,
-                         sprintf("Rscript -e \'args <- commandArgs(TRUE)\' -e \'install.packages(args[1], dependencies=TRUE)\' %s", package),
-                         ";")
+getJobPackageInstallationCommand <- function(type, packages){
```

Comment:
Do we run into issues doing this for each job rather than the cluster? I assume we get the benefit of job-prep running as same user as task.